client/my-sites/themes/controller.jsx

```diff
@@ -24,6 +24,7 @@ function getProps( context ) {
 	const { basePath, analyticsPageTitle } = getAnalyticsData(
 		context.path,
 		tier,
+		filter,
```

Comment:
Nice spot, not sure what I was thinking here. Will revert this, tracking in #7193.